tensorflow/go/op/scope_test.go

```diff
@@ -95,6 +95,40 @@ func TestMultipleGeneratedOps(t *testing.T) {
 	}
 }
 
+func TestScopeWithGraph(t *testing.T) {
+	s1 := NewScope()
+
+	var hello interface{} = "hello"
+	tensor1, _ := tf.NewTensor(hello)
+	s1.AddOperation(tf.OpSpec{
```

Comment:
That's because of the name collission. Perhaps you can replace the second `Const` with:

```go
s2 := NewScopeWithGraph(graph)
Const(s2.SubScope("addition"), "world")
...
```